feature-libs/cart/base/components/add-to-cart/add-to-cart.component.ts

```diff
@@ -144,6 +159,19 @@ export class AddToCartComponent implements OnInit, OnDestroy {
 
     this.addToCartForm.controls['quantity'].setValue(1);
 
+    if (this.featureToggles.showRealTimeStockInPDP) {
```

Comment:
MAJOR: do you think we have to put the code in line 175 and 180 inside else block of this IF?

Reply:
using that is causing the PDP to initially show out of stock for a second and then shows the available quantity.